src/patternfly/components/Table/templates/table--scrollable--tbody--tr.hbs

```diff
@@ -0,0 +1,19 @@
+{{#> table-tr}}
+  {{#if table--scrollable--Column1IsStickyColumn}}
+    {{> table--scrollable--th table--scrollable--th--modifier=table--scrollable--th--modifier--cell-1-modifier table--scrollable--th--content=cell-1--content table-th--IsStickyColumn="true" table--scrollable--NoIcon="true"}}
+  {{else}}
+    {{> table--scrollable--td table--scrollable--td--content=cell-1--content}}
+  {{/if}}
+  {{#if table--scrollable--Column2IsStickyColumn}}
+    {{> table--scrollable--th table--scrollable--th--modifier=table--scrollable--th--modifier--cell-2-modifier table--scrollable--th--content=cell-2--content table-th--IsStickyColumn="true" table-th--attribute='style="--pf-c-table__sticky-column--MinWidth: 200px; --pf-c-table__sticky-column--Left: 200px;"'}}
```

Comment:
`--pf-c-table__sticky-column--MinWidth: 200px` is the default. Do you want to use that here? Or do you want to set that to some other value on a column to show how it changes the left value of adjacent columns?

Reply:
Updated this to illustrate `MinWidth` and `Left` relationship